JobConfig/recoOnSpill.fcl

```diff
@@ -0,0 +1,19 @@
+#
+# Reconstruct OnSpill events. This only reconstructs LoopHelices
+#
+#include "Production/JobConfig/reco/Reco.fcl
```

Comment:
Missing " at end of this line. Is this file supposed to be at reco/OnSpill.fcl?

Reply:
This file was also misnamed, thanks for catching this. It is now JobConfig/reco/OnSpill.fcl, and is functional